android/app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,7 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".MainActivity"
+            android:theme="@style/AppTheme"
```

Comment:
I believe this is redundant -- we already have it on the application element, above.